ai_review/algo/utils.py

```
@@ -0,0 +1,1248 @@
+
+def _fix_key_value(key: str, value: str):
+    key = key.strip().upper()
+    value = value.strip()
+    try:
+        value = yaml.safe_load(value)
+    except Exception as e:
+        get_logger().debug(f"Failed to parse YAML for config override {key}={value}", exc_info=e)
+    return key, value
+
+
+def load_yaml(response_text: str, keys_fix_yaml: List[str] = [], first_key="", last_key="") -> dict:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)